### PR TITLE
improve: Update alt attributes for images on the front page

### PR DESCRIPTION
**Pull request by MeenuyD**
Lots of the images on the front page of the Kubernetes website (https://kubernetes.io/) have unhelpful alt attributes, such as the filename. This commit addresses this issue by updating the alt attributes with the empty string.
#42832

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
Thank you for your contribution! Just noticed that most of the images now have empty alt attributes, which is great. However, there are still a few images remaining that seem to be generated from the `'case-studies'` shortcode. To address this, you can navigate to the ['case-studies.html'](https://github.com/kubernetes/website/blob/main/layouts/shortcodes/blocks/case-studies.html#L16) file _(specifically to line 16)_ and make a similar change. Once that's done, we should be all set. 
Also, you can verify the changes through the Preview Link - https://deploy-preview-43088--kubernetes-io-main-staging.netlify.app/ .

**Comment by MeenuyD** [comment]
Thank you, @dipesh-rawat for your guidance and assistance! Your help has been great.

File: layouts/shortcodes/blocks/case-studies.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h3>{{ $caseStudiesSection.Title }}</h3>
 				{{ $logo = .Resources.GetMatch "**logo*.png" }}
 			{{ end }}
       <div>
-        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="{{ .Title }}">{{ end }}
+        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="">{{ end }}
```
Comment (tengqm) [inline layouts/shortcodes/blocks/case-studies.html:16]: I believe this alt attributes are there to improve accessibility.

Comment (MeenuyD) [reply layouts/shortcodes/blocks/case-studies.html:16]: ok

**Comment by tengqm** [comment]
/close
This change is irrelevant.

**Comment by tengqm** [comment]
/reopen
Okay, we do have an issue filed.

**Pull request merged**